src/test/java/redis/clients/jedis/tests/commands/PublishSubscribeCommandsTest.java

```diff
@@ -302,8 +336,8 @@ public void onPMessage(byte[] pattern, byte[] channel, byte[] message) {
   }
 
   @Test
-  public void binarySubscribeLazily() throws UnknownHostException, IOException,
```

Comment:
@marcosnils  will you mind if I run the "make format" and fix all the formatting in one go ? Because if I do that then we will be seeing some formatting changes to approximately 30 existing files.